src/lib/cht-session.ts

```diff
@@ -15,23 +15,22 @@ axiosRetry(axios, axiosRetryConfig);
 export default class ChtSession {
   public readonly authInfo: AuthenticationInfo;
   public readonly username: string;
-  public readonly facilityId: string;
+  public readonly facilityIds: string[];
   public readonly axiosInstance: AxiosInstance;
   public readonly sessionToken: string;
 
-  private constructor(authInfo: AuthenticationInfo, sessionToken: string, username: string, facilityId: string) {
+  private constructor(authInfo: AuthenticationInfo, sessionToken: string, username: string, facilityIds: string[]) {
```

Comment:
won't this be a `string` and not a `string[]` on older versions of cht-core?

Reply:
my point about the test stands please. it's hard to keep these scenarios straight without regression tests